### PR TITLE
update: - The language change button has been updated in terms of UX …

### PR DESCRIPTION
**Pull request by mouafus**
- [x] The language change button has been updated in terms of UX and positioning on the homepage and in the drawer
- [x] Changing the page from the drawer no longer refreshes the browser tab;
- [x] page 'add-report', we can now open the drawer without any chance of getting the "Cannot POST /add-alert" error
- [x] error during the build of the "report" icon which was not loaded fixed
- [x]  Translation of drawer contents, add-report page and other contents

This will solve #141

### CODE REVIEW

File: frontend/src/components/drawer/Drawer.component.tsx [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 import menu from "../../assets/img/menu.png";
+import {useTranslation} from "react-i18next";
+import {Box} from "@mui/material";
+import {useHistory} from "react-router-dom";
 
 export const MyDrawer = (props: any) => {
+    const {t, i18n} = useTranslation();
     const [isOpen, setIsOpen] = useState(false);
     const toggleDrawer = () => {
+        console.log("problem herer");
```
Comment (Asam237) [inline frontend/src/components/drawer/Drawer.component.tsx:13]: Remove the console.log

**Review by Asam237 — CHANGES_REQUESTED** [review]

**Comment by Asam237** [comment]
Great work @mouafus

File: frontend/src/components/drawer/Drawer.component.tsx (hunk 1 above)
Comment (mouafus) [reply frontend/src/components/drawer/Drawer.component.tsx]: Done

**Review by mouafus — COMMENTED** [review]
I removed that `console.log`

**Review by Asam237 — APPROVED** [review]

**Pull request merged**